projects/packages/forms/src/contact-form/class-contact-form-endpoint.php

```diff
@@ -653,4 +659,31 @@ public function get_akismet_status() {
 			)
 		);
 	}
+
+	/**
+	 * Get Jetpack CRM plugin status and extension data.
+	 *
+	 * @return WP_REST_Response Plugin status data.
+	 */
+	public function get_jetpack_crm_status() {
+		$plugin_status = $this->get_plugin_status( 'jetpack-crm' );
+		$status_data   = $plugin_status->get_data();
+
+		if ( $status_data['isActive'] ) {
+			// @phan-suppress-next-line UndefError
+			$has_extension = function_exists( 'zeroBSCRM_isExtensionInstalled' ) && zeroBSCRM_isExtensionInstalled( 'jetpackforms' );
```

Comment:
Phan gives an function undefined error on this line, even though the function is global, and we're checking function_exists() before using it. 

```
projects/packages/forms/src/contact-form/class-contact-form-endpoint.php:673
Error: UndefError PhanUndeclaredFunction Call to undeclared function zeroBSCRM_isExtensionInstalled()
```

Known issue - see [here](https://github.com/phan/phan/issues/1204). We could also work around this by adding a stub, but that seems excessive.